src/main/kotlin/me/partlysanestudios/partlysaneskies/PartlySaneSkies.kt

```diff
@@ -149,29 +149,40 @@ class PartlySaneSkies {
             private set
 
         // Names of all the ranks to remove from people's names
-        val RANK_NAMES = arrayOf(
-            "[VIP]", "[VIP+]", "[MVP]", "[MVP+]", "[MVP++]", "[YOUTUBE]", "[MOJANG]",
-            "[EVENTS]", "[MCP]", "[PIG]", "[PIG+]", "[PIG++]", "[PIG+++]", "[GM]", "[ADMIN]", "[OWNER]", "[NPC]"
-        )
+        val RANK_NAMES =
+            arrayOf(
+                "[VIP]",
+                "[VIP+]",
+                "[MVP]",
+                "[MVP+]",
+                "[MVP++]",
+                "[YOUTUBE]",
+                "[MOJANG]",
+                "[EVENTS]",
+                "[MCP]",
+                "[PIG]",
+                "[PIG+]",
+                "[PIG++]",
+                "[PIG+++]",
+                "[GM]",
+                "[ADMIN]",
+                "[OWNER]",
+                "[NPC]",
+            )
```

Comment:
Not relevent to this pr but I feel like this isn't the right location for it haha